docs/en/sql-reference/functions/tuple-map-functions.md

```diff
@@ -62,7 +62,7 @@ Alias: `MAP_FROM_ARRAYS(keys, values)`
 
 **Arguments**
 
-- `keys` —  Array of keys to create the map from. [Array(T)](../data-types/array.md) where `T` can be any type supported by [Map](../data-types/map.md) as key type.
+- `keys` —  Array or map of keys to create the map from. [Array(T)](../data-types/array.md) where `T` can be any type supported by [Map](../data-types/map.md) as key type, or [Map](../data-types/map.md).
```

Comment:
l. 46: "Creates a map from an array or map of keys and an array or map of values."

Reply:
L. 88 has an additional example for maps as 2nd argument. It would be helpful to add yet another example to the docs for maps as 1st argument (after l. 100).